contributors.md

```diff
@@ -1,5 +1,5 @@
 Contributors
 ============
 
-bananas24 - teacher
-HaggardCoder -Eduardo
+   bananas24 - teacher
```

Comment:
I'd like the spaces after the name, rather than at the beginning. Something like:
``
bananas24    - teacher
``
``
HaggardCoder - Eduardo
``